proj/kws_micro_accel/src/tensorflow/lite/kernels/internal/reference/integer_ops/kws_conv.h

```diff
@@ -0,0 +1,216 @@
+/* Copyright 2021 The CFU-Playground Authors
```

Comment:
If it's not a huge amount of effort, could you please add this file with the unoptimized op in one commit, then in a second commit, make the optimizing changes. This will help future readers understand the change.

I realize that this may not be easily possible, and if so, don't worry about it.

Reply:
Unfortunately these changes have been in different scratch directories for so long that I would probably have to redo the PR to build some reasonable ordering. Happily, as you mentioned, the software optimizations aren't too obscure, really just constant folding (some of which the compiler does automatically).

This is great feedback that I didn't consider though (I hadn't thought about how commit history acts as documentation), certainly I'll keep in mind more fine-grained commits for any future work.